### PR TITLE
Player SFX Version 1

### PR DESCRIPTION
**Pull request by DelaineWTan**

### CODE REVIEW

File: ImmunityReborn/Assets/_TeamAmnesia/Scripts/State Machine/Player/PlayerUsingHealState.cs
Comment (chrisyandev) [inline ImmunityReborn/Assets/_TeamAmnesia/Scripts/State Machine/Player/PlayerUsingHealState.cs:1]: What's the purpose of usedHeal variable? Player uses heal only when animation reaches a certain point, and exits this state immediately.

File: ImmunityReborn/Assets/_TeamAmnesia/Scripts/Combat/HealthConsumable.cs
Comment (chrisyandev) [inline ImmunityReborn/Assets/_TeamAmnesia/Scripts/Combat/HealthConsumable.cs:1]: What was the issue with playing the ParticleSystem?

File: ImmunityReborn/Assets/_TeamAmnesia/Scripts/State Machine/Player/PlayerUsingHealState.cs
Comment (DelaineWTan) [reply ImmunityReborn/Assets/_TeamAmnesia/Scripts/State Machine/Player/PlayerUsingHealState.cs:1]: Ah the player was healing a bunch of times every cast but it was a bug unrelated to this and this variable is actually unnecessary. Good catch! Removed.

File: ImmunityReborn/Assets/_TeamAmnesia/Scripts/Combat/HealthConsumable.cs
Comment (DelaineWTan) [reply ImmunityReborn/Assets/_TeamAmnesia/Scripts/Combat/HealthConsumable.cs:1]: Not an issue necessarily but I wanted to play the sound while the particle system plays then destroy the whole object when done. We no longer need a particle system on the player prefab itself as a result and this is consistent with how VFX/SFX is played in the rest of the game.

**Review by chrisyandev — APPROVED** [review]
Nice

**Pull request merged**